packages/charting/src/components/LineChart/LineChart.styles.ts

```diff
@@ -33,28 +33,56 @@ export const getStyles = (props: ILineChartStyleProps): ILineChartStyles => {
     ],
     xAxis: [
       {
-        transform: `translate(${xOffset}px, ${height}px)`
+        transform: `translate(-${width * scalingVal}px, ${height}px) scaleX(1.5)`
       }
     ],
     xAxisTicks: [],
     yAxis: [
       {
-        transform: `translate(${yOffset}px, 0px)`
+        transform: `translate(-${width * scalingVal}px, 0px)`
       }
     ],
     yAxisTicks: [
       {
-        transform: 'scaleX(-1)'
+        transform: `scaleX(-${width * scalingVal - xOffset})`,
+        stroke: 'grey'
       }
     ],
     yAxisDomain: [
       {
-        transform: 'scaleX(-1)'
+        transform: 'scaleX(0)'
+      }
+    ],
+    xAxisDomain: [
+      {
+        transform: 'scaleY(0)'
+      }
+    ],
+    xAxisText: [
+      {
+        transform: `translate(0px,13px)`
       }
     ],
     lines: [
       {
-        transform: `translate(${xOffset}px, 0px)`
+        transform: `translate(-${width * 0.3}px, 0px) scaleX(1.5)`
+      }
+    ],
+    legendContainer: [
+      {
+        marginTop: '10px',
+        paddingLeft: '5%'
+      }
+    ],
+    calloutPadding: [
+      {
+        padding: '10px 16px 10px 16px',
+        backgroundColor: palette.white,
+        fontSize: '28px',
```

Comment:
Font-related properties (fontSize, fontWeight, and fontFamily) should come from theme.fonts (e.g. theme.fonts.large).